config/template.toml

```diff
@@ -3,14 +3,18 @@ json_rpc_address = "127.0.0.1:3030"
 
 [subnets]
 
-[subnets.root]
+[[subnets]]
 id = "/root"
+network_name = "root"
+gateway_addr = 64
 jsonrpc_api_http = "https://example.org/rpc/v0"
 jsonrpc_api_ws = "wss://example.org/rpc/v0"
 auth_token = "YOUR ROOT AUTH TOKEN"
 
-[subnets.child]
+[[subnets]]
```

Comment:
From total ignorance, would it be possible to make the config something like so that we use the `id` already as the key of the map?: 
```
[subnets]

[[/root]]
....
[[/root/t01011]]
```

Reply:
This one I'm not so sure, but I think it cannot be done, because `[[/root]]` would translate to hashmap of "/root" as key and whatever in the `...` as an array of deserialized struct. But I think we should avoid using subnet id string because of `/`, everytime we have to escape it with `"`, which is easy to make mistake.